test/verify/machineslib.py

```diff
@@ -1815,6 +1828,7 @@ def installWithErrorTest(dialog):
 
             # Disable selinux because it makes TFTP directory inaccesible and we don't want sophisticated configuration for tests
             self.machine.execute("if type selinuxenabled >/dev/null 2>&1 && selinuxenabled; then setenforce 0; fi")
+            self.addCleanup(self.machine.execute, "setenforce 1 || true")
```

Comment:
That looks worrying. How exactly did it break? It's idempotent, i. e. if it's already in enforcing, the command still succeeds. So if re-enabling it fails, doesn't that mean the machine is already broken? Is there some way to reproduce that scenario? I have a feeling we need to clean up harder here.

Also, I want to get rid of this completely, see issue #13717. I think I should get to that today.

Reply:
```
root@debian:~# setenforce 1
setenforce: SELinux is disabled
root@debian:~# echo $?
1
```
It should be wrapped in `if type selinuxenabled >/dev/null 2>&1 && selinuxenabled; then setenforce 1; fi` as we disable selinux, but this was a quick workaround as there is #13717 to resolve this properly anyway.

Reply:
Running `testCreate` locally is a nuisance, but I manually did the TFTP server steps and left SELinux enabled. I originally expected that I have to do something like `chcon -t svirt_t` to the created directories, but I can create a working PXE boot VM just like that. Perhaps you can try to just  drop the setenforce 0 bit completely?

Reply:
But doing what you said is ok as well. We can then try and drop it completely in a separate PR, if you prefer.